okhttp/src/main/java/okhttp3/internal/http/RetryAndFollowUpInterceptor.kt

```diff
@@ -59,71 +59,68 @@ class RetryAndFollowUpInterceptor(private val client: OkHttpClient) : Intercepto
     var followUpCount = 0
     var priorResponse: Response? = null
     while (true) {
-      call.prepareExchangeFinder(request)
-
-      if (call.isCanceled()) {
-        throw IOException("Canceled")
-      }
+      call.enterNetworkInterceptorExchange(request)
 
       var response: Response
-      var success = false
+      var closeActiveExchange = true
       try {
```

Comment:
This diff goes from `try {} catch {} finally {}` to `try { try {} catch } finally {}`. It is easier to see with whitespace changes hidden.